style.css

```diff
@@ -329,8 +473,68 @@ body.lock {
         display: none;
     }
     .content {
-        padding: 50px 0 0;
+        padding: 0 0;
     } 
+    .carousel__text {
+        top: 15%;
+        left: 3%;
+    }
+    .carousel {
+        position: static; 
+        width: 100vw;
+        height: auto;
+        overflow: visible;
+    }
+    .carousel__box {
+        position: static;
+        display: inline;
+    }
+    .carousel__object {
+        width: 32%;
+        margin: 0 1px;
+    }
+    .carousel__buttons {
+        display: none;
+    }
+    .about-part {
+        display: flex; 
+        flex-direction: row;
+        height: auto;
+    }
+    .about-part__img-container_img1{
+        left: 10%;
+    }
+    .about-part__img-container_img2 {
+        right: 10%; 
+    }
+    .about-part__img-container {
+        flex: 1 0;
+    }
+    .about-part__text-container {
+        width: 50%;
+    }
+    .title {
+        font-size: 35px;
+    }
+    .gallery-part {
+        background-color: var(--light-gray-color);
+        text-align: center;
+        padding: 40px 0;
+    }
+    .gallery-part__photos {
+        margin: 20px 5px;
+        display: inline;
```

Comment:
Why not use flexbox?

Reply:
I use grid in the mobile version, but in the desktop version it is enough display: inline;